ios/sandwich/QonversionSandwich.swift

```diff
@@ -75,6 +75,26 @@ public class QonversionSandwich : NSObject {
     Qonversion.shared().purchase(productId, completion: purchaseCompletion)
   }
   
+  @objc public func purchase(
+    _ productId: String,
+    quantity: Int,
+    contextKeys: [String],
+    completion: @escaping BridgeCompletion
+  ) {
+    let purchaseCompletion = getPurchaseCompletionHandler(for: completion)
+    let purchaseOptions = Qonversion.PurchaseOptions(quantity: quantity, contextKeys: contextKeys)
```

Comment:
Small optimization - maybe worth moving these lines below the product existence check?